content/menu.md

```diff
@@ -298,6 +298,20 @@ site_menu:
   - category: Streamlit library / API reference / Personalization / st.experimental_user
     url: /library/api-reference/personalization/st.experimental_user
     isVersioned: false
+  - category: Streamlit library / API reference / Connections and databases
+    url: /library/api-reference/connections
+  - category: Streamlit library / API reference / Connections and databases / st.experimental_connection
+    url: /library/api-reference/connections/st.experimental_connection
+    isVersioned: true
+  - category: Streamlit library / API reference / Connections and databases / SQLconnection
+    url: /library/api-reference/connections/st.connections.sqlconnection
+    isVersioned: true
+  - category: Streamlit library / API reference / Connections and databases / Snowparkconnection
```

Comment:
typos on these ('c' is lowercase)

`SQLconnection` -> `SQLConnection`

`Snowparkconnection` -> `SnowparkConnection

Reply:
Thank you! Fixed on 1a49042